scalafix-tests/input/src/main/scala/test/LeakingSealed.scala

```diff
@@ -0,0 +1,18 @@
+/*
+rules = LeakingSealed
```

Comment:
Can we add a test case where LeakingSealed and DisableSyntax.noNonFinalCaseClass are both enabled? It seems they might conflict, any idea how we could consolidate them?

Reply:
hmm, `noNonFinalCaseClass` fixes the `LeakingSealed` problem, can we run it in fixed order?

Reply:
I would recommend baking the two into one rule, something like `MissingFinal`. It can later be extended to add `final` to `def` that are not overridden etc.

Reply:
Should it add `final` to regular classes when they extend sealed things? And should it ignore traits since they can't be `final`?

Reply:
@olafurpg what do you think?